ui/css/perf.css

```diff
@@ -449,3 +449,23 @@ td.alert-confidence {
 .alert-strike {
     text-decoration: line-through;
 }
+
+.compareTableTooltip .tooltip-inner {
+  max-width: 440px;
+}
+
+.toolTipTable {
+  border: 1px solid white;
+  height: 20px;
+  margin-bottom: 5px;
+}
+.toolTipTable .valueColumn {
+  width: 50px;
+  border: 1px solid white;
+  align: center;
+}
+
+.toolTipTable .distributionColumn{
+  width: 350px;
```

Comment:
Let's reduce the width here a bit as this makes the tooltips quite wide -- could you try a value of 280?